model/team.go

```diff
@@ -40,7 +40,7 @@ type Team struct {
 	LastTeamIconUpdate int64   `json:"last_team_icon_update,omitempty"`
 	SchemeId           *string `json:"scheme_id"`
 	GroupConstrained   *bool   `json:"group_constrained"`
-	PolicyID           *string `json:"policy_id" db:"-"`
+	PolicyId           *string `json:"policy_id"`
```

Comment:
We cannot change the field name unfortunately as that's a breaking change. Let's keep `PolicyID`